model-optimizer/extensions/front/tf/ObjectDetectionAPI.py

```diff
@@ -1147,6 +1147,16 @@ def generate_sub_graph(self, graph: Graph, match: SubgraphMatch):
         conv_nodes = backward_bfs_for_operation(detection_output_node.in_node(0), ['Conv2D'])
         swap_weights_xy(graph, conv_nodes)
 
+        # As outputs are replaced with a postprocessing node, outgoing tensor names are no longer
+        # correspond to original tensors and should be removed from output->Result edges
+        for out_idx in range(match.outputs_count()):
+            out_node = match.output_node(out_idx)[0]
+            if out_node.out_edges():
+                new_fw_info = []
+                for fw_info in out_node.out_edge(0)['fw_tensor_debug_info']:
+                    new_fw_info.append((fw_info[0], fw_info[1], None))
+                out_node.out_edge(0)['fw_tensor_debug_info'] = new_fw_info
```

Comment:
We have here again an issue with output port number as in the file above. 

Also, let's move this code into a separate function like "get_attribute_between_nodes(node1, node2, attr_name)" and "set_attribute_between_nodes(node1, node2, attr_name)" and use it in model-optimizer/extensions/front/tf/ObjectDetectionAPI.py and model-optimizer/extensions/front/tf/RetinaNetFilteredDetectionsReplacement.py and model-optimizer/extensions/front/output_cut.py